### DISCUSSION TITLE
How can we customize search bar when using search delegates?

### DISCUSSION THREAD

**Discussion by bijayyas** [community/community#28056] [Mobile] [answered]
How can we customize search bar when using search delegates?

**Comment by stevepopovich** [comment]
Hi @bijayyas, can you please clarify exactly what you are looking for here within the GitHub Mobile app?

**Comment by surajadkhari** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/48079501/182814907-9731ae07-89d1-46a7-a0ba-20e5d6b56a19.png)
 We are trying to customize the searchDelegate:
-You can customize by using defining theme particular for searchDelegate class:
![image](https://user-images.githubusercontent.com/48079501/182815697-a4e34004-e09e-4d38-beb4-e9100af6b9b7.png)
